node/src/cli/commands/login.command.ts

```diff
@@ -69,6 +70,10 @@ export class LoginCommand {
     let clientId: string = null;
     let clientSecret: string = null;
 
+    let captchaClientSecret: string = null;
+
+    let selectedProvider: any = null;
```

Comment:
Hmm - I'm a bit nervous to adjust that since its a bit outside the scope of my changes. Would feel more comfortable tackling that outside of this PR.

Reply:
I think it's a pretty safe change, but we can leave it for the refactor.